app/main.py

```diff
@@ -212,6 +216,24 @@ def get_website_verification(appid: str):
     return verification.get_website_verification(appid)
 
 
+@app.post("/generate-download-token", status_code=200)
+def get_download_token(appids: List[str]):
```

Comment:
Could we please at least have this gatekept with a login, and a TODO to say "we need to check for rights here" ?